binding/java/test/ai/picovoice/rhino/RhinoTest.java

```diff
@@ -23,30 +23,66 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.HashMap;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 public class RhinoTest {
 
+    private static final String ENVIRONMENT_NAME;
     private Rhino rhino;
     private String accessKey = System.getProperty("pvTestingAccessKey");
 
-    private static String getTestContextPath() {
-        String environmentName = Utils.getEnvironmentName();
-        return String.format(
-                "../../resources/contexts/%s/coffee_maker_%s.rhn",
-                environmentName,
-                environmentName
-        );
+    static {
+        ENVIRONMENT_NAME = Utils.getEnvironmentName();
+    }
+
+    private static String append_language(String s, String language) {
+        if (language == "en")
+            return s;
+        return s + "_" + language;
+    }    
+
+    private static String getContextPath(String language, String context) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../resources")
+            .resolve(append_language("contexts", language))
+            .resolve(ENVIRONMENT_NAME)
+            .resolve(context + "_" + ENVIRONMENT_NAME + ".rhn")
+            .toString();
     }
 
-    @BeforeEach
-    void setUp() throws RhinoException {
+    private static String getModelPath(String language) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../lib/common")
+            .resolve(append_language("rhino_params", language)+".pv")
+            .toString();
+    }    
+
+    private static String getAudioFilePath(String audioFileName) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../resources/audio_samples")
+            .resolve(audioFileName)
+            .toString();
+    }
+
+    private void buildRhinoWrapper(String language, String context) throws RhinoException{
+        rhino = new Rhino(
```

Comment:
same as my feedback on porcupine - just create the instances of Rhino using the Builder interface in each test case